<launch>
    <include file="$(find astra_camera)/launch/astra.launch" />

    <node name="yolo_onnx_trt" pkg="sign_detect" type="yolo-benchmark" output="screen" args="-o /visualsign">
        <param name="batchSize" value="1"/>
        <param name="channel" value="3"/>
        <param name="height" value="288"/>
        <param name="width" value="384"/>
        <param name="nbCls" value="6"/>
        <param name="engineFile" value="data/3l.trt"/>
        <param name="keepTopK" value="1"/>
        <param name="keepTopK" value="1"/>
    </node>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find sign_detect)/rviz/sign_visualize.rviz" />
</launch>